classes/test.php

```diff
@@ -1181,6 +1181,12 @@ public function runTestMethod($testMethod, array $tags = array())
 						)
 					;
 
+					$this->factoryBuilder->build($testedClass)
+						->addToAssertionManager($this->assertionManager, 'callStaticOnTestedInstance', function($methodName) use ($testedClass) {
```

Comment:
@mikaelrandy the factory builder is used to build a factory which will help atoum instanciate tested class.

Because we don't know in advance how to instanciate the tested class, we had to write a builder to dynamically create factories. 

This is the only goal of the factory builder ;)

In your case, you don't have to instanciate the tested class because you just want to use static calls.

_related to your question on IRC_